target/linux/lantiq/image/vr9.mk

```diff
@@ -96,6 +96,36 @@ define Device/arcadyan_vgv7519-nor
 endef
 TARGET_DEVICES += arcadyan_vgv7519-nor
 
+define Device/arcadyan_vgv952cjw33-e-ir
+  $(Device/NAND)
+  SOC := vr9
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := VGV952CJW33-E-IR
+  DEVICE_ALT0_VENDOR := Vodafone
+  DEVICE_ALT0_MODEL := Easybox 904xDSL
+  SUPPORTED_DEVICES += VGV952CJW33-E-IR
```

Comment:
This line is not required if you add a new device.

Reply:
I have write up, exist a description what are the meaning of the different variables and which consequenses have there change ?

Reply:
None that I know of. I typically just google for them (combined with openwrt keyword) and hope to find a helpful wiki entry. Otherwise, I have to dig into the repo itself.